src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -468,11 +467,11 @@ bool InputProcessor::checkForUnsupportedObjects(EnergyPlusData &state)
 
 bool InputProcessor::processErrors(EnergyPlusData &state)
 {
-    auto const idf_parser_errors = idf_parser->errors();
-    auto const idf_parser_warnings = idf_parser->warnings();
+    auto const &idf_parser_errors = idf_parser->errors();
+    auto const &idf_parser_warnings = idf_parser->warnings();
 
-    auto const validation_errors = validation->errors();
-    auto const validation_warnings = validation->warnings();
+    auto const &validation_errors = validation->errors();
+    auto const &validation_warnings = validation->warnings();
```

Comment:
Why not just remove these?

    for (auto const &error : idf_parser->errors()) {
        ShowSevereError(state, error);
    }

Reply:
Could do that, thx. This was a fairly "mindless" pass, I didn't count uses or anything like that.